.golangci.yml

```diff
@@ -43,6 +43,28 @@ linters:
   - whitespace
 
 linters-settings:
+  revive:
```

Comment:
Since the slow performance of revive is tracked as an issue, maybe add a comment to re-enable them once a future release improves performance?